pyris/api/extract.py

```diff
@@ -122,6 +123,19 @@ def get_iris_field(code, limit=None, geojson=False):
     return res
 
 
+def get_iris_list_by_city_code(code):
```

Comment:
Is is possible to add an optional `geojson` parameter like the other functions? In that way, it will be possible to use the `geojson_parser` or the `address_parser` as mentioned previoulsy. And the URL will look like `URL/api/city/code/33063?geojson=true` or `URL/api/city/search?q=QUERY&geoson=true`.

I think It can be done in another PR.